setup.py

```diff
@@ -73,7 +73,7 @@ def gen_data_files(*dirs):
     "cycler",
     "pyyaml",
 ]
-tests_require = ["pytest", "pytest-cov", "pytest-mock"]
+tests_require = ["pytest", "pytest-cov", "pytest-mock", "yabox"]
```

Comment:
Well, this is not really a requirement only needed in the tests, but actually to run the optmizer, so it shuold be in either in the `install_requires` above or as a subtype of `extras_require`:
```python
extras_require = {"dev": tests_require + ["Sphinx", "pre-commit"], "opt": ["yabox"]}
```